src/Types/Rational.purs

```diff
@@ -43,11 +43,11 @@ derive newtype instance Semiring Rational
 derive newtype instance Ring Rational
 derive newtype instance CommutativeRing Rational
 
-instance EncodeJson Rational where
-  encodeJson r = encodeJson
+instance EncodeAeson Rational where
```

Comment:
We need a DecodeAeson for this as well...

Reply:
Don't we have one?